src/batches/batches.tsx

```diff
@@ -186,15 +186,30 @@ const BatchesComponent = (): React.JSX.Element => {
     setDeletePopupOpen(false);
   };
 
-  const renderActions = (data: { name: string }) => {
+  const renderActions = (data: {
+    state: BatchStatus; name: string 
+}) => {
     return (
       <div className="actions-icon" role="button" aria-label="Delete Job">
         <div
-          className="icon-buttons-style"
+         className={
+          data.state === BatchStatus.STATUS_PENDING
+            ? 'icon-buttons-style-delete-batch-disable'
+            : 'icon-buttons-style-delete-batch'
+         }
           title="Delete Batch"
-          onClick={() => handleDeleteBatch(data.name.split('/')[5])}
+          onClick={
+            data.state === BatchStatus.STATUS_PENDING
```

Comment:
move the BatchStat.STATUS_PENDING check into  handleDeleteBatch

Reply:
resolved